core/Dockerfile

```diff
@@ -44,13 +42,16 @@ RUN dpkgArch="$(dpkg --print-architecture | awk -F- '{ print $NF }')"; \
     ./gosu --version && \
     ./gosu nobody true
 
-FROM ${base_image}
+##############################################################################
+######################### Build for RedHat OpenShift #########################
+##############################################################################
+
+FROM registry.access.redhat.com/ubi9/ubi AS rhel
```

Comment:
Can you explain this magic, please? What the QuestDB docker image will be based on at the end?

Reply:
See the updated description for more detail